pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -351,6 +351,8 @@ func (ds *Plugin) UpdateAttestedNode(ctx context.Context, n *common.AttestedNode
 		if err != nil {
 			return err
 		}
+		// TODO: this is at the wrong level of the software stack.
+		// It should be created in the caller of the datastore interface.
```

Comment:
I think we missed removal of this TODO. Can you please remove?

Reply:
Fixed in next push.